userguide/advanceduse/ssh.rst

```diff
@@ -22,6 +22,9 @@ You need then to transfer your public key to your device. There are multiple way
 * You can also connect via :doc:`adb <adb>` and use the following command to copy it::
 
     adb push ~/.ssh/id_rsa.pub /home/phablet/
+    
+Please note:Some of the UT devices might not deteched on connecting through USB or adb devices might not list your device,In such situations transfering the key via the internet to your phone is a better approcah.
+
```

Comment:
Doesn't the sentence above "Or transfer ..." clarify this already?

Reply:
I had seen a couple of queries in the telegram group around not being able to connect through adb. Users could be of the impression that adb would work even in UT as they would have connected to the same phone through adb in android.
Mentioning it explicitly in dcoumentation would perhaps reduce the confusion.

Reply:
Ok. I can imagine there is *some* way to make it more explicit, but in the proposed form it looks really strange. 

> Do this thing. There are a few options:
> * do A 
> * do B 
> * do C

and now we add a sentence at the bottom:

> Oh and btw if you have trouble with option C, why not try option A.

Reply:
I removed the statement.Please review the rest.